code/modules/clothing/gloves/color.dm

```diff
@@ -90,7 +90,18 @@
 	item_color = "hosred"		//Exists for washing machines. Is not different from black gloves in any way.
 
 /obj/item/clothing/gloves/color/black/ce
-	item_color = "chief"			//Exists for washing machines. Is not different from black gloves in any way.
+	desc = "These stylish black gloves are both insulated and offer protection from heat sources."
+	name = "engineering gloves"
+	item_color = "chief"        // exists for washing machines
+	siemens_coefficient = 0
+	permeability_coefficient = 0.05
+	strip_delay = 80
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	resistance_flags = NONE
```

Comment:
You don't need these lines, 99-103, because the parent `/obj/item/clothing/gloves/color/black` has these set.

Reply:
Good catch!